docs/usage/self-hosted-configuration.md

```diff
@@ -214,6 +214,13 @@ The order method for autodiscover server side repository search.
 
 The sort method for autodiscover server side repository search.
 
+Platform supported sort options:
+
+| Platform       | Supported sort options                      |
+| -------------- | ------------------------------------------- |
+| GitLab         | `created_at`, `updated_at`, `id`            |
+| Forgejo, Gitea | `alpha`, `created`, `updated`, `size`, `id` |
+
```

Comment:
Nice! In the future, I think we should look at if we can provide better config validation to make sure folks find out that i.e. `alpha` isn't supported when set on GitLab